matching-service/routes/matchRoutes.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+const router = express.Router()
+import * as matchController from '../controllers/matchController.js'
+
+router.post('/find-match', matchController.findMatch)
+router.delete('/cancel-find-match', matchController.cancelFindMatch)
+router.get('/interview/:id', matchController.getInterview)
+router.delete('/end-interview/:id', matchController.deleteInterview)
```

Comment:
Referring to line 127 and 137 in matchController.js should be "interview/:interviewId"  and "/end-interview/:interviewId" since should be req.params.interviewId in there as well